.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
 1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
 1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
+</details>
+
+<details>
+<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. `ssh <USER.NAME>@hasta.scilifelab.se`
+1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
+1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
+1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
+1. Make sure the branch is deployed: `us; scout --version`
+1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
```

Comment:
Sorry, it just came to my mind. perhaps another step here that illustrates how to test? example:

scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml <command>

Reply:
whoops, too late. No worries, another time!

Reply:
Or say it's a servers PR in that case. 😸